backend/src/main/java/com/redhat/cloud/notifications/routers/models/SettingsValueByEventTypeJsonForm.java

```diff
@@ -33,6 +33,7 @@ public static class Field {
         public String checkedWarning;
         @JsonInclude(Include.NON_NULL)
         public String infoMessage;
+        public boolean disabled;
```

Comment:
Why is this field named disabled?

Reply:
I mean, I know that the settings values code is complex but

Reply:
This field will be used in the frontend to disable an HTML component (checkbox).

Reply:
Ah, and I guess it has to be named like that. All right! Thanks!

Reply:
I have no idea how it will be used in the frontend 😅 but `disabled` didn't sound like a bad name so I went with that one. We can easily change it to anything later.